cognite/client/data_classes/files.py

```diff
@@ -214,6 +217,23 @@ def remove(self, value: List) -> FileMetadataUpdate:
     # GenStop
 
 
+# GenPropertyClass: FilesAggregate
+class FilesAggregate(dict):
```

Comment:
```suggestion
class FileAggregate(dict):
```
need too rename for consistency

Reply:
the name comes from the [schema](https://storage.googleapis.com/cognitedata-api-docs/dist/v1.json), I just left it without the changes

Reply:
schemas are not user facing and can be changed, the name we choose here is more permanent

Reply:
got it, will change